test/parallel/test-https-timeout.js

```diff
@@ -21,6 +21,7 @@
 
 'use strict';
 const common = require('../common');
+const fixtures = require('../common/fixtures');
```

Comment:
@	mikemfleming - on a second look, I guess this require statement can move down the line, after the hasCrypto check.